recipe/conda_build_config.yaml

```diff
@@ -70,27 +72,29 @@ flann:
 fontconfig:
   - 2.12
 freetype:
-  - 2.8                # [not ppc64le]
+  - 2.8.1              # [not ppc64le]
 gstreamer:
   - 1.12
 gst_plugins_base:
   - 1.12
+gdal:
+  - 2.2.*
 geos:
   - 3.6.2
 giflib:
   - 5.1
 glib:
-  - 2.53
+  - 2.55
 glpk:
-  - 4.62
+  - 4.61
 gmp:
-  - 6
+  - 6.1
```

Comment:
`6` is enough. Minor versions add symbols, but doesn't remove them. So, `run_exports` with a minor version as the min would fix this.

Reply:
Ok, done.